### PR TITLE
add some glitter to code of conduct page

### PR DESCRIPTION
**Pull request by christiansherry**

### CODE REVIEW

File: views/about.twig [hunk 1]
```diff
@@ -5,171 +5,182 @@
...
+    </div>
+    <div class="wrapper clearfix document">
 
-            <h3><a name="talks">Talks</a></h3>
+        <h3><a name="talks">Talks</a></h3>
 
-            <p>Most of our meetings have talks from community members and we’re always looking for people to give a talk.
-            If you’d like to give a talk you should get in touch with us (@phpdorset, phpdorset@gmail.com) and suggest it.</p>
+        <p>Most of our meetings have talks from community members and we’re always looking for people to give a talk.
+        If you’d like to give a talk you should get in touch with us (@PHP Dorset, PHP Dorset@gmail.com) and suggest it.</p>
```
Comment (fullybaked) [inline views/about.twig]: At a guess, your find and replace to put a space in PHPDorset has affected the Twitter handle and Email address

File: views/about.twig [hunk 2]
```diff
@@ -5,171 +5,182 @@
...
                 <ul>
-                    <li>behave respectfully towards other people — they’re real humans, just like you</li>
-                    <li>be considerate of people’s time and attention</li>
-                    <li>contribute positively and constructively to discussions (online or in-person)</li>
+                    <li><a href="#talks">Talks</a>
+                        <ul>
+                            <li><a href="#scheduling">Scheduling</a></li>
+                            <li><a href="#subject-matter">Subject matter</a></li>
+                            <li><a href="#timing">Timing</a></li>
+                            <li><a href="#after-talks">After the talks</a></li>
```
Comment (fullybaked) [inline views/about.twig]: After talks in the sub-menu here, but made into it's own section in the content.  Should it either be included in the Talks section or the menu item put to the top level to maintain consistency in the document structure

File: views/about.twig (hunk 1 above)
Comment (christiansherry) [inline views/about.twig]: Sorry Dave, this was lazy of me!

File: views/about.twig [hunk 3]
```diff
@@ -5,171 +5,179 @@
...
-            <p>In return for your sponsorship PHPDorset will:
-                <ul>
-                    <li>put your logo in the sponsor side-bar on the relevant meeting page on phpdorset.co.uk</li>
-                    <li>mention the specifics of your sponsorship in the main details of the relevant meeting page on phpdorset.co.uk</li>
-                    <li>mention the sponsorship in the meeting announcement email on the mailing list</li>
-                    <li>tweet about the sponsorship from @phpdorset at least once to announce it and once on the day as a reminder</li>
-                    <li>mention you at the start of the meeting, and make sure you have 30seconds or so during our announcements section to talk to the group</li>
-                </ul>
-            </p>
+        <p>There are opportunities for drinks sponsorship during the meeting, or bar-tab sponsorship after the meeting at the pub. If you would like to provide sponsorship you should get in touch with us (@PHPDorset, PHPDorset@gmail.com) to find out the prices and arrange it.</p>
```
Comment (rossey) [inline views/about.twig]: `@PHPDorset, PHPDorset@gmail.com` - should be URLs

**Pull request merged**